app/controllers/api/proposals_controller.rb

```diff
@@ -34,8 +35,19 @@ def show
     render json: @proposal
   end
 
+  def update
```

Comment:
Also I'm pretty sure this `can? :update, Proposal` has an effect on other parts of the system (maybe `ProposalsController` has an update too?)